### PR TITLE
Svelte 4 #14

### PR DESCRIPTION
**Pull request by brandonp-ais**
Updates project and peerDependencies to Svelte 4.

Migration done by running `npx svelte-migrate@latest svelte-4` from svelte migration document linked in issue.

### CODE REVIEW

**Comment by labriola** [comment]
It's been a couple of months. Any concern on moving forward with this?

**Comment by bondib** [comment]
+1 on that

File: package.json [hunk 1]
```diff
@@ -1,70 +1,70 @@
...
+		"test": "playwright test",
+		"check": "svelte-check --tsconfig ./tsconfig.json",
+		"check:watch": "svelte-check --tsconfig ./tsconfig.json --watch",
+		"lint": "prettier --check --plugin-search-dir=. . && eslint .",
+		"format": "prettier --write --plugin-search-dir=. .",
+		"release": "commit-and-tag-version"
+	},
+	"repository": {
+		"type": "git",
+		"url": "git+https://github.com/brandonp-ais/ag-grid-svelte.git"
```
Comment (ddxv) [inline package.json:49]: @brandonp-ais just curious, shouldn't these stay pointing to MichaelKim's repo for the PR?

**Review by ddxv — COMMENTED** [review]
@brandonp-ais just looking at the PR out of curiosity, was curious about why the packagelock and changelog changed to your repo. There are 2-3 spots that this changed, which I think if it's for your own main fork that's ok, but seemed odd for merging back into the parent?

File: package.json (hunk 1 above)
Comment (brandonp-ais) [reply package.json:49]: I don't know tbh. This PR was poorly done on my part and I'll close it. But the process for updating to svelte-4 is outlined in the issue.

**Comment by brandonp-ais** [comment]
PR makes incorrect changes to package.json. Better done by maintainer or someone else.

**Comment by labriola** [comment]
For anyone stumbling upon this, I would likely consider this project dead.

**Pull request closed without merging**